packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -116,6 +117,34 @@ export function addBlobToSummary(
 	summary.stats.totalBlobSize += getBlobSize(content);
 }
 
+export function addBlobToSummaryViaPath(
```

Comment:
This function needs comments with example. It is not at all clear how it adds the given blob to the summary.

Reply:
It also needs unit tests. I can see this easily breaking.